statuspage/src/locales/pl/translation.json

```diff
@@ -0,0 +1,36 @@
+{
+  "common": {
+    "poweredBy": "Stworzony przez",
+    "refresh": "Odświerz"
+  },
+  "timespan": {
+    "1year": "1 rok",
+    "6months": "6 miesięcy",
+    "1month": "1 miesiąc",
+    "1week": "1 tydzień",
+    "1day": "1 dzień"
+  },
+  "status": {
+    "availability": "Dostępność",
+    "availabilityWarning": "Availability warning",
+    "availabilityError": "Availability error",
+    "lowAvailabilityText": "We're currently experiencing reduced availability of {{percent}} % in the last 30 minutes.",
+    "latency": "Opóźnienie",
+    "latencyWarning": "Latency warning",
+    "latencyError": "Latency error",
+    "highLatencyText": "We're currently experiencing unusually high latency of {{ms}} ms in the last 30 minutes.",
```

Comment:
Any chance to also get a translation for these? :)

Reply:
Done. Thx